kubeval/1.0.0/README.md

```diff
@@ -0,0 +1,40 @@
+# Kubeval Execution
+
+Use kubeval to validate Kubernetes YAML files inside a Keptn sequence. More info on [kubeval.com](https://kubeval.com)
+
+Whenever valid files are provided, the Keptn task passes:
+
+![valid evaluation](assets/valid.png)
```

Comment:
As described in https://github.com/keptn-contrib/artifacthub/pull/104 Artifacthub currently doesn't support relative paths. The reason is that when processing packages AH only reads the metadata files. They don't store or serve any other files. Therefore all the relative paths in the PR should be changed to absolute paths. Will also add this fact to the README to avoid future problems.